app/src/main/java/com/example/cosc345project/workers/IndexingWorker.kt

```diff
@@ -27,7 +27,7 @@ class IndexingWorker @AssistedInject constructor(
         setForeground(createForegroundInfo())
 
         try {
-            searchRepository.indexFromFirebase()
+//            searchRepository.indexFromFirebase()
```

Comment:
This needs to be changed, as I think it was a leftover test when the emulators weren't working